pkg/controller/account/account_controller.go

```diff
@@ -1256,6 +1256,21 @@ func (r *ReconcileAccount) handleCreateAdminAccessRole(
 			return nil, nil, err
 		}
 
+		_, err = createManagedOpenShiftSupportRole(
+			reqLogger,
+			awsSetupClient,
+			ccsClient,
+			adminAccessArn,
+			currentAccInstanceID,
+			tags,
+			SREAccessARN,
```

Comment:
In the case of these roles, we won't be able to use the SREAccessARN the same way we're using it now, we'll need to add the STS access ARN to the configmap and use that.

Essentially, right now we're using a role on the payer account called like CCS-Access or something, which is different than the one we're using for STS jump-role access.  We need to get the STS access arn and add that to App Interface as a parameter, and update the deployed configmap to have that ARN, and then use the same STS jump role here instead of the SREAccessARN.

This is probably confusing as hell, so please reach out if it doesn't make sense.